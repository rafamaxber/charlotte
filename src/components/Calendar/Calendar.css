.DayPicker {
  font-family: 'Heebo', sans-serif;
  font-style: normal;
  font-weight: normal;
  line-height: normal;
  font-size: 15.8px;
  text-align: center;
  letter-spacing: 1.79545px;
  color: #555555;
}
.DayPicker-Weekday {
  color: #B5B5B5;
  font-size: 15px;
}

.DayPicker-Day {
  padding: 11px;
}

.DayPicker-NavBar {
  display: none;
}
.DayPicker-Caption {
  text-align: center;
  font-size: 15px;
}

.DayPicker-WeekdaysRow {
  border-bottom: 2px solid #E3E3E3;
  padding: 10px 0;
}

.DayPicker-Day--selected:not(.DayPicker-Day--disabled):not(.DayPicker-Day--outside):hover {
  background-color: transparent;
}

.Selectable .DayPicker-Day--selected:not(.DayPicker-Day--start):not(.DayPicker-Day--end):not(.DayPicker-Day--outside) {
  background-color: #79BD1A !important;
  color: #fff;
}

.Selectable .DayPicker-Day {
  border-radius: 0 !important;
  font-size: 15px;
  color: #555555;
}

.DayPicker:not(.DayPicker--interactionDisabled) .DayPicker-Day:not(.DayPicker-Day--disabled):not(.DayPicker-Day--selected):not(.DayPicker-Day--outside):hover {
  background-color: transparent;
  position: relative;
}

.DayPicker-Caption>div {
  font-size: 15px;
}

.DayPicker:not(.DayPicker--interactionDisabled) .DayPicker-Day:not(.DayPicker-Day--disabled):not(.DayPicker-Day--selected):not(.DayPicker-Day--outside):hover:after {
  content: "";
  position: absolute;
  display: block;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  z-index: -1;
  transition: .3s ease;
  background-color: rgba(200, 255, 124, 0.356);
  border-radius: 50%;

}

.Selectable .DayPicker-Day--start {
  background-color: #79BD1A;
  border-top-left-radius: 50%;
  border-bottom-left-radius: 50%;
}

.DayPicker-Week {
  border: 10px solid #fff;
}

.Selectable .DayPicker-Day.DayPicker-Day--start {
  border-top-left-radius: 50%;
  border-bottom-left-radius: 50%;
  background-color: transparent;
  z-index: 10;
  color: #fff;
}

.Selectable .DayPicker-Day.DayPicker-Day--end {
  border-top-right-radius: 50%;
  border-bottom-right-radius: 50%;
  background-color: transparent;
  z-index: 10;
  color: #79BD1A;
}

.Selectable .DayPicker-Day.DayPicker-Day--end:after {

  content: "";
  height: 100%;
  width: 100%;
  border: 2px solid #79BD1A;
  position: absolute;
  background-color: #fff;
  top: 0;
  left: 0;
  border-radius: 50%;
  z-index: -1;
}

.Selectable .DayPicker-Day.DayPicker-Day--start:before {
  content: "";
  border-top-left-radius: 50%;
  border-bottom-left-radius: 50%;
  background-color: #79BD1A;
  position: absolute;
  top: 0;
  left: 0;
  height: 100%;
  z-index: -1;
  width: 100%;
}


.Selectable .DayPicker-Day.DayPicker-Day--end:before {
  content: "";
  border-top-right-radius: 50%;
  border-bottom-right-radius: 50%;
  background-color: #79BD1A;
  position: absolute;
  top: 0;
  left: 0;
  height: 100%;
  z-index: -1;
  width: 100%;
}

