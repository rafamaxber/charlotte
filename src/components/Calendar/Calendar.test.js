import React from 'react';
import renderer from 'react-test-renderer';
import Calendar from './Calendar';

describe('Calendar', () => {
  it('Should match with snapshot', () => {
    const component = renderer.create(
      <Calendar></Calendar>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
