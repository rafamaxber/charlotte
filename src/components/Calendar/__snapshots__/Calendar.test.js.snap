// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar Should match with snapshot 1`] = `
<div
  className="WrapperCalendar"
>
  <div
    className="DayPicker Selectable"
    lang="en"
  >
    <div
      className="DayPicker-wrapper"
      onBlur={undefined}
      onFocus={undefined}
      onKeyDown={[Function]}
      tabIndex={-1}
    >
      <div
        className="DayPicker-NavBar"
      >
        <span
          aria-label="Previous Month"
          className="DayPicker-NavButton DayPicker-NavButton--prev"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex="0"
        />
        <span
          aria-label="Next Month"
          className="DayPicker-NavButton DayPicker-NavButton--next"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex="0"
        />
      </div>
      <div
        className="DayPicker-Months"
      >
        <div
          className="DayPicker-Month"
          role="grid"
        >
          <div
            className="DayPicker-Caption"
            role="heading"
          >
            <div
              onClick={undefined}
              onKeyUp={[Function]}
            >
              February 2018
            </div>
          </div>
          <div
            className="DayPicker-Weekdays"
            role="rowgroup"
          >
            <div
              className="DayPicker-WeekdaysRow"
              role="row"
            >
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Sunday"
                >
                  Su
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Monday"
                >
                  Mo
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Tuesday"
                >
                  Tu
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Wednesday"
                >
                  We
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Thursday"
                >
                  Th
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Friday"
                >
                  Fr
                </abbr>
              </div>
              <div
                className="DayPicker-Weekday"
                role="columnheader"
              >
                <abbr
                  title="Saturday"
                >
                  Sa
                </abbr>
              </div>
            </div>
          </div>
          <div
            className="DayPicker-Body"
            role="rowgroup"
          >
            <div
              className="DayPicker-Week"
              role="row"
            >
              <div
                aria-disabled={true}
                className="DayPicker-Day DayPicker-Day--outside"
                style={undefined}
              />
              <div
                aria-disabled={true}
                className="DayPicker-Day DayPicker-Day--outside"
                style={undefined}
              />
              <div
                aria-disabled={true}
                className="DayPicker-Day DayPicker-Day--outside"
                style={undefined}
              />
              <div
                aria-disabled={true}
                className="DayPicker-Day DayPicker-Day--outside"
                style={undefined}
              />
              <div
                aria-disabled={false}
                aria-label="Thu Feb 01 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={0}
              >
                1
              </div>
              <div
                aria-disabled={false}
                aria-label="Fri Feb 02 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                2
              </div>
              <div
                aria-disabled={false}
                aria-label="Sat Feb 03 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                3
              </div>
            </div>
            <div
              className="DayPicker-Week"
              role="row"
            >
              <div
                aria-disabled={false}
                aria-label="Sun Feb 04 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                4
              </div>
              <div
                aria-disabled={false}
                aria-label="Mon Feb 05 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                5
              </div>
              <div
                aria-disabled={false}
                aria-label="Tue Feb 06 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                6
              </div>
              <div
                aria-disabled={false}
                aria-label="Wed Feb 07 2018"
                aria-selected={false}
                className="DayPicker-Day DayPicker-Day--today"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                7
              </div>
              <div
                aria-disabled={false}
                aria-label="Thu Feb 08 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                8
              </div>
              <div
                aria-disabled={false}
                aria-label="Fri Feb 09 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                9
              </div>
              <div
                aria-disabled={false}
                aria-label="Sat Feb 10 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                10
              </div>
            </div>
            <div
              className="DayPicker-Week"
              role="row"
            >
              <div
                aria-disabled={false}
                aria-label="Sun Feb 11 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                11
              </div>
              <div
                aria-disabled={false}
                aria-label="Mon Feb 12 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                12
              </div>
              <div
                aria-disabled={false}
                aria-label="Tue Feb 13 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                13
              </div>
              <div
                aria-disabled={false}
                aria-label="Wed Feb 14 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                14
              </div>
              <div
                aria-disabled={false}
                aria-label="Thu Feb 15 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                15
              </div>
              <div
                aria-disabled={false}
                aria-label="Fri Feb 16 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                16
              </div>
              <div
                aria-disabled={false}
                aria-label="Sat Feb 17 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                17
              </div>
            </div>
            <div
              className="DayPicker-Week"
              role="row"
            >
              <div
                aria-disabled={false}
                aria-label="Sun Feb 18 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                18
              </div>
              <div
                aria-disabled={false}
                aria-label="Mon Feb 19 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                19
              </div>
              <div
                aria-disabled={false}
                aria-label="Tue Feb 20 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                20
              </div>
              <div
                aria-disabled={false}
                aria-label="Wed Feb 21 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                21
              </div>
              <div
                aria-disabled={false}
                aria-label="Thu Feb 22 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                22
              </div>
              <div
                aria-disabled={false}
                aria-label="Fri Feb 23 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                23
              </div>
              <div
                aria-disabled={false}
                aria-label="Sat Feb 24 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                24
              </div>
            </div>
            <div
              className="DayPicker-Week"
              role="row"
            >
              <div
                aria-disabled={false}
                aria-label="Sun Feb 25 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                25
              </div>
              <div
                aria-disabled={false}
                aria-label="Mon Feb 26 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                26
              </div>
              <div
                aria-disabled={false}
                aria-label="Tue Feb 27 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                27
              </div>
              <div
                aria-disabled={false}
                aria-label="Wed Feb 28 2018"
                aria-selected={false}
                className="DayPicker-Day"
                onClick={[Function]}
                onFocus={undefined}
                onKeyDown={[Function]}
                onMouseDown={undefined}
                onMouseEnter={undefined}
                onMouseLeave={undefined}
                onMouseUp={undefined}
                onTouchEnd={undefined}
                onTouchStart={undefined}
                role="gridcell"
                style={undefined}
                tabIndex={-1}
              >
                28
              </div>
              <div
                aria-disabled={true}
                className="DayPicker-Day DayPicker-Day--outside"
                style={undefined}
              />
              <div
                aria-disabled={true}
                className="DayPicker-Day DayPicker-Day--outside"
                style={undefined}
              />
              <div
                aria-disabled={true}
                className="DayPicker-Day DayPicker-Day--outside"
                style={undefined}
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
